modules/EmailTemplates/EmailTemplateParser.php

```diff
@@ -81,7 +81,7 @@ class EmailTemplateParser
      * @param Campaign $campaign
      * @param User $user
      */
-    public function __construct(EmailTemplate $template, Campaign $campaign, User $user)
```

Comment:
Why are we changing the signature here? Do we need to update the PHP Doc block to accurate depict what types the $user should be?